src/lib/core/typography/_typography.scss

```diff
@@ -42,17 +42,21 @@
 }
 
 // Adds the base typography styles, based on a config.
-@mixin mat-typography($config: mat-typography-config(), $selector: '.mat-typography') {
+// TODO(crisbeto): the headers need to handle margins as well.
+@mixin mat-base-typography($config, $selector: '.mat-typography') {
   .mat-h0, .mat-hero-header {
     @include mat-typography-level-to-styles($config, display-4);
+    letter-spacing: -0.05em;
```

Comment:
Ah I see, it seems it switched from Arial to Roboto which explains the difference, maybe we should add `.mat-h4/5` to the element so it looks a little better?